src/background/deinflect.ts

```diff
@@ -517,6 +537,18 @@ export function deinflect(word: string): CandidateWord[] {
         // Continue if the rule introduces a duplicate in the reason chain,
         // as it wouldn't make sense grammatically.
```

Comment:
Now that the `continue;` is further down, I think we need to update this comment (or simply move it down near the `continue;`).